### PR TITLE
Cleanup old things from sitemap

### PR DESCRIPTION
**Pull request by rhyolight**

### CODE REVIEW

**Comment by rhyolight** [comment]
@lscheinkman Removed things from sitemap we don't want.

File: packages/numenta.org/pages/sitemap/index.jsx [hunk 1]
```diff
@@ -112,16 +112,6 @@ const SitemapPage = (props, {config, route}) => {
...
-                <TextLink to={links.in.papers}>
-                  Research Papers
-                </TextLink>
-              </ListItem>
-              <ListItem>
-                <TextLink to={links.in.faq}>
-                  Frequently Asked Questions (FAQ)
-                </TextLink>
-              </ListItem>
-              <ListItem>
```
Comment (lscheinkman) [inline packages/numenta.org/pages/sitemap/index.jsx]: Are `links.in.papers` and `links.in.faq` used anywhere else? Shouldn't they be also removed from [config.toml](https://github.com/numenta/numenta-web/blob/master/packages/numenta.org/config.toml#L31)

Comment (rhyolight) [inline packages/numenta.org/pages/sitemap/index.jsx]: Done thanks

**Comment by rhyolight** [comment]
I also added a test snapshot I forgot to add to source control when I moved the section.

**Comment by lscheinkman** [comment]
You forgot this link:

https://github.com/rhyolight/numenta-web/blob/2a8a248ee7007e402ce813dd317e8f4a606275a1/packages/numenta.org/pages/sitemap/index.jsx#L173-L181

**Comment by lscheinkman** [comment]
And this category
https://github.com/rhyolight/numenta-web/blob/2a8a248ee7007e402ce813dd317e8f4a606275a1/packages/numenta.org/pages/sitemap/index.jsx#L49

**Comment by lscheinkman** [comment]
👍 LGTM

**Pull request merged**